### PR TITLE
Add ReplicatedData to EnsembleSpectrum

### PR DESCRIPTION
**Pull request by etyley**

### CODE REVIEW

File: sbnana/CAFAna/XSec/Flux.cxx [hunk 1]
```diff
@@ -130,12 +130,13 @@ namespace ana
     std::unique_ptr<TVectorD> pdg((TVectorD*)dir->Get("pdg"));
     assert(pdg && pdg->GetNrows()==1);
 
-    return std::unique_ptr<FluxTimesNuclei>(new FluxTimesNuclei(Spectrum::LoadFrom(dir, "spectrum"), (*pdg)[0]));
+    return std::unique_ptr<FluxTimesNuclei>(
+        new FluxTimesNuclei(std::move(*Spectrum::LoadFrom(dir, "spectrum")), (*pdg)[0]));
```
Comment (cjbackhouse) [inline sbnana/CAFAna/XSec/Flux.cxx:134]: Is this `move()` really necessary? It feels like it shouldn't be (the unique_ptr from LoadFrom is already a temporary)

File: sbnana/CAFAna/XSec/Flux.cxx [hunk 2]
```diff
@@ -173,6 +174,14 @@ namespace ana
   {
   }
 
+  //----------------------------------------------------------------------
+  EnsembleFluxTimesNuclei EnsembleFluxTimesNuclei::ReplicatedNominal(const FluxTimesNuclei& spec,
+                                                                  const FitMultiverse* multiverse)
```
Comment (cjbackhouse) [inline sbnana/CAFAna/XSec/Flux.cxx:179]: Alignment

File: sbnana/CAFAna/XSec/Flux.cxx [hunk 3]
```diff
@@ -233,12 +242,13 @@ namespace ana
     std::unique_ptr<TVectorD> pdg((TVectorD*)dir->Get("pdg"));
     assert(pdg && pdg->GetNrows()==1);
 
-    return std::unique_ptr<EnsembleFluxTimesNuclei>(new EnsembleFluxTimesNuclei(EnsembleSpectrum::LoadFrom(dir, "ensemblespectrum"), (*pdg)[0]));
+    return std::unique_ptr<EnsembleFluxTimesNuclei>(
+        new EnsembleFluxTimesNuclei(std::move(*EnsembleSpectrum::LoadFrom(dir, "ensemblespectrum")), (*pdg)[0]));
```
Comment (cjbackhouse) [inline sbnana/CAFAna/XSec/Flux.cxx:246]: Same question about move() here

**Review by cjbackhouse — APPROVED** [review]
This is fine, with a few minor comments.

File: sbnana/CAFAna/XSec/Flux.cxx (hunk 1 above)
Comment (etyley) [reply sbnana/CAFAna/XSec/Flux.cxx:134]: The compiler complains that it is an lvalue if you just dereference the unique_ptr

**Pull request merged**